gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/explode/ExplodeBundle.java

```diff
@@ -49,6 +50,8 @@ void explodeBundle(String folderPath, FilterConfiguration filterConfiguration, F
         BundleFilter bundleFilter = ExportPluginModule.getInjector().getInstance(BundleFilter.class);
         Bundle filteredBundle = bundleFilter.filter(folderPath, filterConfiguration, bundle);
 
+        bundleFilter.validateImportedFolder(folderPath, filteredBundle);
```

Comment:
is this the right place for doing this?
I mean, some lines above you have the full bundle loaded with no filtering, so you could check before if the folder path is valid and stop there if not, then you avoid extra processing that will be useless.